MdeModulePkg/Core/Dxe/Misc/MemoryProtectionSupport.c

```diff
@@ -1232,184 +1234,194 @@ GetOverlappingMemorySpaceRegion (
 
 /**
   Updates the memory map to contain contiguous entries from StartOfAddressSpace to
-  max(EndOfAddressSpace, address + length of the final memory map entry). Gaps in the
-  input EFI memory map which correlate with the non-existent GCD type will not be added
-  to the map.
-
-  @param[in, out] MemoryMapSize         Size, in bytes, of MemoryMap
-  @param[in, out] MemoryMap             IN:  Pointer to the EFI memory map which will have all gaps filled. The
-                                             original buffer will be freed and updated to a newly allocated buffer
-                                        OUT: A sorted memory map describing the entire memory region
-  @param[in]      DescriptorSize        Size, in bytes, of each descriptor region in the array
-  @param[in]      StartOfAddressSpace   Starting address from which there should be contiguous entries
-  @param[in]      EndOfAddressSpace     Ending address at which the memory map should at least reach
+  max(EndOfAddressSpace, address + length of the final memory map entry). If DetermineSize
+  is TRUE, then this function will just determine the required buffer size for the output
+
+  memory map.

```

Comment:
Clean up empty line.

```suggestion
  is TRUE, then this function will just determine the required buffer size for the output
  memory map.
```